src/extension.ts

```diff
@@ -132,6 +133,7 @@ export async function activate(context: vscode.ExtensionContext) {
   if (config.get<boolean>('semanticHighlighting'))
     semanticHighlighting.activate(client, context);
   client.registerFeature(new EnableEditsNearCursorFeature);
+  new typeHierarchy.TypeHierarchyProvider(context, client);
```

Comment:
We use the "activate" pattern for new features, could you follow it (see `semanticHighlighting.activate` for reference)?